src/test/java/fr/paris/lutece/portal/service/daemon/DaemonSchedulerTest.java

```diff
@@ -281,26 +287,27 @@ private void testSchedule( boolean shouldThrow ) throws ClassNotFoundException,
             testDaemon.waitForCompletion( );
             assertTrue( testDaemon.hasRun( ) );
         }
+        catch(  InterruptedException | BrokenBarrierException | TimeoutException e)
+        {
+            fail("failed" + e + ":" + e.getMessage( ) );
```

Comment:
here you swallow the exception and you loose the stack trace which might be helpful. Why not let tests throw exceptions, which makes them fail ?